test/_globalHooks.js

```diff
@@ -1,6 +1,7 @@
 import lockfile from 'lockfile';
 
-afterEach(function globalAfterEach() {
+/* eslint-disable mocha/no-top-level-hooks */
+afterEach(function globalAfterEach(done) {
```

Comment:
We're avoiding callbacks as far as possible. Can we use `return`s in this file instead?